### PR TITLE
Nt navigtion steps

### PR DESCRIPTION
**Pull request by nooralnabi**

### CODE REVIEW

File: app/controllers/ads_controller.rb [hunk 1]
```diff
@@ -72,14 +86,24 @@ def edit_prov_select
   end
 
   def map
+    if params[:gender].present?
+      @gender_select = params[:gender]
+      @age_max_select = params[:age_max]
+      @age_min_select = params[:age_min]
+    end 
+    @selected_prov = params[:prov]
     $gender = [[t('views.ad.new.all'), 'All'],[t('views.ad.new.male'), 'Male'], [t('views.ad.new.female'), 'Female']]
```
Comment (MwMaki) [inline app/controllers/ads_controller.rb]: @nooralnabi 
Do you mean to use @ instead of $  in ($gender)

File: app/controllers/ads_controller.rb [hunk 2]
```diff
@@ -91,13 +115,27 @@ def edit_map
...
```
Comment (MwMaki) [inline app/controllers/ads_controller.rb]: Here as well

File: app/views/ads/map.html.haml [hunk 3]
```diff
@@ -23,7 +88,11 @@
                 .g-mb-10.g-px-30
                   %label.g-color-gray-dark-v2.g-font-weight-700.g-font-size-16
                     = t('views.address.new.gender')
-                  = t.select :gender, $gender, {}, class: " form-control g-color-black g-bg-white g-bg-white--focus g-brd-gray-light-v4 g-brd-orange--hover rounded g-px-15"
+                  -if @gender_select.present?
+                    = t.select :gender,options_for_select($gender, selected: @gender_select) ,{}, class: " form-control g-color-black g-bg-white g-bg-white--focus g-brd-gray-light-v4 g-brd-orange--hover rounded g-px-15"
+                  -else
+                    = t.select :gender, $gender,{}, class: " form-control g-color-black g-bg-white g-bg-white--focus g-brd-gray-light-v4 g-brd-orange--hover rounded g-px-15"
```
[the remark this answers is not in the document]
Comment (nooralnabi) [reply app/views/ads/map.html.haml]: oh ok I fixed that because that used by kandelchy

[the remark this answers is not in the document]
Comment (MwMaki) [reply app/views/ads/map.html.haml]: Please change it and we will change it to @ in everywhere later

**Pull request merged**